### PR TITLE
add loglevel controller

### PR DESCRIPTION
**Pull request by deads2k**
This controller allows dynamically changing the log level based on our operator resource

/hold

holding for proof

### CODE REVIEW

**Comment by deads2k** [comment]
this worked locally

File: pkg/operator/loglevel/util.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package loglevel
+
+import operatorv1 "github.com/openshift/api/operator/v1"
+
+func LogLevelToGlog(logLevel operatorv1.LogLevel) int {
```
Comment (mfojtik) [inline pkg/operator/loglevel/util.go]: nit: Klog

**Comment by mfojtik** [comment]
unit test need update

otherwise LGTM

**Comment by deads2k** [comment]
ooms

/retest

**Pull request merged**